v3/cypress/support/e2e.ts

```diff
@@ -22,6 +22,9 @@ import "./commands"
 // add code coverage support
 import "@cypress/code-coverage/support"
 
+// add support for dispatching native events (https://github.com/dmtrKovalenko/cypress-real-events)
+import "cypress-real-events"
+
```

Comment:
Oh, is this what got attribute button clicks working in the tests?

Reply:
I added that as part of getting cypress tests of the new formula editor to work, so I don't think it's related to the commented out tests.